sos/plugins/networking.py

```diff
@@ -188,9 +195,6 @@ def setup(self):
             "bridge -d vlan show"
         ])
 
-        if self.get_option("traceroute"):
-            self.add_cmd_output("/bin/traceroute -n %s" % self.trace_host)
-
```

Comment:
Removed by mistake? :)

Reply:
Removed by mistake, new commit have this option again.